slacker/main.py

```diff
@@ -52,6 +55,19 @@ def check():
       sys.exit(-1)
   slacker_logger.info('All good: {} valid commands'.format(reg.count()))
 
+def build_prompt_completer():
+  """ Build REPL completion dictionary """
```

Comment:
Is it a PEP8 thing with the spaces before and after? I saw it in the other lib also.